pcdsdevices/interface.py

```diff
@@ -899,25 +904,29 @@ def set_position(self, position):
         self.set_current_position(position)
 
 
-def setup_preset_paths(**paths):
+def setup_preset_paths(**kwargs):
```

Comment:
Nitpick: marginally simpler
```suggestion
def setup_preset_paths(defer_loading: bool = False, **paths):
```